rpmpatch/SpecFile.py

```diff
@@ -65,7 +66,7 @@ def build(self, define_dict=None, compileit=False, build_arches=None):
 
         defines = []
         if isinstance(define_dict, dict):
-            for key in define_dict.keys():
+            for key in list(define_dict.keys()):
```

Comment:
Próbowałem znaleźć różnicę między iteracją po `dict.keys()`,  `list(dict.keys())`, a `list(dict)`, ale we wszystkich przypadkach jest to poprawne. 
Natomiast `list(dict.keys())` **wydaje się** najbardziej skomplikowana i najmniej efektywna

Reply:
fixed